src/components/NavBar/style.tsx

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+import { LinkColored } from '@/styles/text';
+
+export const NavBarContainer = styled.div`
+  /* top: 0;
+  left: 0;
+  position: absolute; */
+  width: 100%;
+  height: 78px;
+  background: ${COLORS.blueMid};
+  align-items: center;
```

Comment:
we can probably get away with getting rid of `align-items` here since you're already doing this in `NavBarSectionDiv` (previously `CustomDIv`)